guides/common/modules/ref_operating-system-requirements.adoc

```diff
@@ -1,37 +1,44 @@
 :_mod-docs-content-type: REFERENCE
 
-[id="supported-operating-systems_{context}"]
-= Supported operating systems
+[id="operating-system-requirements"]
+= Operating system requirements
 
-ifdef::satellite[]
-You can install the operating system from a disc, local ISO image, Kickstart, or any other method that Red{nbsp}Hat supports.
-Red{nbsp}Hat {ProductName} is supported on the latest version of {RHEL} 9 that is available at the time when {ProductName} is installed.
-Previous versions of {RHEL} including EUS or z-stream are not supported.
+ifdef::foreman-deb[]
+The following operating systems are supported for deploying {Project}:
 endif::[]
-
-The following operating systems are supported by the installer, have packages, and are tested for deploying {Project}:
-
-.Operating systems supported by {foreman-installer}
-|====
-| Operating System | Architecture | Notes
-ifdef::foreman-el,katello,orcharhino[]
-| {PlanningDocURL}Enterprise_Linux[{EL}] 9 | x86_64 only | EPEL is not supported.
+ifndef::foreman-deb[]
+The following operating system is supported for deploying {Project}:
```

Comment:
Just a suggestion (non-blocking): Katello-builds currently only run on EL9. In the future, they might run on EL10 too. It looks like you have preprared for this eventuality by using a list.

However, you'd have to reword the introduction - at least for the grace period in which F+K runs on both EL9 and EL10 - from "operating system" to "operating systems".

As an alternative, we could just say `You run deploy and run {Project} on {EL} 9 (x86_64).`. You call; I am OK either way.

Reply:
I like how the bullet point highlights the most important part of the module, even if it really is a one-item list. However, you have a point about the need to update the intro in the future. So what I decided to do is move the bullet points into the same ifdef block as the introductions. It's not much but it might help whoever ends up updating the list in the future to also notice the introduction.